fastfood/food.py

```diff
@@ -42,33 +41,23 @@ def build_cookbook(build_config, templatepack, cookbooks_home,
     with open(build_config) as cfg:
         cfg = json.load(cfg)
 
+    cookbook_name = cfg['name']
     cookbook = None
-    if cookbook_path:
-        try:
-            cookbook = book.CookBook(cookbook_path)
-        # change this to a more specific error
-        except ValueError:
-            # create a new one please
-            pass
+    cookbook_path = os.path.join(cookbooks_home, cookbook_name)
```

Comment:
Good point. I'll remove the argument as well. The idea here is that the target is always going to be `os.path.join(cookbook_directory,cookbook name)`, so there's no point in specifying one.